### PR TITLE
Remove @mdi icons as dependency

### PR DESCRIPTION
**Pull request by 2metres**
**User-Facing Changes**
None

**Description**
Absorb existing `@mdi` icons and remove as a dependency

### CODE REVIEW

File: packages/studio-base/src/components/TimeBasedChart/TimeBasedChartLegend.tsx [hunk 1]
```diff
@@ -70,7 +71,7 @@ export default function TimeBasedChartLegend(props: Props): JSX.Element {
         }
 
         const CheckboxComponent =
-          linesToHide[label] === true ? CheckboxBlankOutlineIcon : CheckboxMarkedIcon;
+          linesToHide[label] === true ? CheckBoxOutlineBlankIcon : CheckBoxIcon;
```
Comment (2metres) [inline packages/studio-base/src/components/TimeBasedChart/TimeBasedChartLegend.tsx:74]: This should just be a checkbox TBH

**Review by defunctzombie — DISMISSED** [review]
Which icon package should we be using - mui? It might be nice to make a package to consolidate our icons into that and only ever use icons from that package.

**Comment by jtbandes** [comment]
What's the point of doing this? Why would we maintain our own svgs vs importing them from a source where we can understand the origin of the icon and get updates for it?

**Comment by 2metres** [comment]
> Which icon package should we be using? 

Currently we have multiple options when it comes to which icon should I use:

`@mui/icons-material` - Only official material UI icons exported as react componets - 177 results, 65 files
`@mdi` Material Design Icons - official and unoffical Material Design Icons - 12 results, 6 files
`@fluentui/react-icons` - Fluent UI system icons (Personally I prefer these to MUI ones) - 39 results, 6 files 
+ a few custom icons we have created for pose, etc.

**Comment by 2metres** [comment]
> What's the point of doing this? Why would we maintain our own svgs vs importing them from a source where we can understand the origin of the icon and get updates for it?

The majority of the icons we are importing from `@mdi` are icons that look like Material Icons but aren't included in the main set for whatever reason. The intent of this PR is not to avocate for importing all the SVG icons we use into our app but to discourage the splintering of our icons. 

Long term I would prefer if we made an effort to use more of the Fluent UI icons as I feel like they are more polished than the Material UI ones. Potentially deleting the embedded ones as well with better alternatives

**Comment by 2metres** [comment]
Reading my own comment I've decided to replace some of the absorbed icons with Fluent UI analogues.

**Comment by jtbandes** [comment]
I don't see any new fluent ui icons being used in the current version of the PR. Forgot to push some changes?

**Review by jtbandes — APPROVED** [review]

**Pull request merged**